gfe-db/pipeline/template.yaml

```diff
@@ -506,7 +506,7 @@ Resources:
       CodeUri: functions/validate_build_output/
       Handler: app.lambda_handler
       Runtime: python3.10
-      MemorySize: 256
+      MemorySize: 1024
       Timeout: 60
```

Comment:
Timeout should be increased to the max, 900 seconds, because this can process multiple releases

Reply:
I also think more memory overhead could be helpful, but one pending change I will make is to move the validation step inside the map step so it's only validating the data for one release. That way it will require less memory and also be able to run concurrently.